dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -16,3 +16,37 @@ function onURLBlocked(base_url, callback) {
       .css("position", "absolute")
       .appendTo("body");
 }
+
+var DateHelper = {
```

Comment:
I might want to merge this and put localizing the timestamp as a P2 (using the plugin may be a bit tricky to integrate because it doesn't happen in our loc plugin)